OnlineFlightSearchAPI/Repositories/IAirportRepository.cs

```diff
@@ -5,6 +5,6 @@ namespace OnlineFlightSearchAPI.Repositories
 {
     public interface IAirportRepository
     {
-        List<AirportDetail> FetchAirportDetail(string airportCode);
+        IEnumerable<AirportDetail> FetchAirportDetail(string airportCode);
```

Comment:
There are no changes in tests due to this?

Reply:
In AirportServicesTests, these are mocked and tested